### ISSUE TITLE
Pivot table cell background color applied on alternate rows only

### ISSUE THREAD

**Issue by HuifangYeo** [atoti/atoti#110]
## Steps to reproduce
I select a default background color for a column:
![image](https://user-images.githubusercontent.com/35210495/87738151-4c656700-c80f-11ea-8bef-70d949fc900c.png)

## Actual Result
The background formatting only applied on alternate rows for pivot and tree table:
![image](https://user-images.githubusercontent.com/35210495/87738189-61da9100-c80f-11ea-8ced-6bbfd1d96b60.png)

Same results in the web application.

## Expected Result
The table format is displayed correctly.
![image](https://user-images.githubusercontent.com/35210495/87738237-7a4aab80-c80f-11ea-9cec-5446a1e0f32d.png)

## Environment
- atoti: 0.4.2.20200713160333
- Operating System: win32

**Comment by HuifangYeo** [comment]
I don't see any plugin configuration that sets background color on alternate rows:
```
{
        "state": {
            "name": "",
            "type": "container",
            "value": {
                "style": {},
                "showTitleBar": false,
                "body": {
                    "serverUrl": "",
                    "mdx": "WITH  Member [Measure Simulations].[Marketing Budget Simulation].[5 x Retention - 10 x Retention] AS [Measure Simulations].[Marketing Budget Simulation].[10 x Retention] - [Measure Simulations].[Marketing Budget Simulation].[5 x Retention], CAPTION = \"Increase in Expense\"    Member [Measures].[Actual Expense (1)] AS [Measures].[Actual Expense], CAPTION = [Measures].[Actual Expense].MEMBER_CAPTION, BACK_COLOR = rgb(159, 197, 232), FORMAT_STRING = \"#,###\"  SELECT NON EMPTY Crossjoin([Measures].[Actual Expense (1)], Union([Measure Simulations].[Marketing Budget Simulation].[Marketing Budget Simulation].Members, [Measure Simulations].[Marketing Budget Simulation].[5 x Retention - 10 x Retention])) ON COLUMNS, NON EMPTY [Epoch].[Epoch].[Branch].Members ON ROWS FROM [customer_cube] CELL PROPERTIES BACK_COLOR, FONT_FLAGS, FORE_COLOR, FORMATTED_VALUE, VALUE",
                    "contextValues": {},
                    "updateMode": "once",
                    "ranges": {
                        "row": {
                            "chunkSize": 2000,
                            "thresholdPercentage": 0.1
                        },
                        "column": {
                            "chunkSize": 50,
                            "thresholdPercentage": 0.2
                        }
                    },
                    "configuration": {
                        "tabular": {
                            "pinnedHeaderSelector": "member",
                            "sortingMode": "non-breaking",
                            "cellRenderers": [
                                "pivot-layout"
                            ],
                            "statisticsShown": true,
                            "columnsGroups": [
                                {
                                    "captionProducer": "firstColumn",
                                    "cellFactory": "kpi-status",
                                    "selector": "kpi-status"
                                },
                                {
                                    "captionProducer": "firstColumn",
                                    "cellFactory": "lookup",
                                    "selector": "lookup"
                                },
                                {
                                    "captionProducer": "expiry",
                                    "cellFactory": "expiry",
                                    "selector": "kpi-expiry"
                                }
                            ],
                            "hideAddButton": true,
                            "columnOrder": {
                                "key": "explicit",
                                "args": {
                                    "orderedColumns": [
                                        "c-treeCells-member",
                                        "([Measures].[Actual Expense (1)],[Measure Simulations].[Marketing Budget Simulation].[Marketing Budget Simulation].[5 x Retention])",
                                        "([Measures].[Actual Expense (1)],[Measure Simulations].[Marketing Budget Simulation].[Marketing Budget Simulation].[10 x Retention])",
                                        "([Measures].[Actual Expense (1)],[Measure Simulations].[Marketing Budget Simulation].[Marketing Budget Simulation].[5 x Retention - 10 x Retention])"
                                    ]
                                }
                            },
                            "defaultOptions": {},
                            "columns": [
                                {
                                    "key": "([Measures].[Actual Expense],[Measure Simulations].[Marketing Budget Simulation].[Marketing Budget Simulation].[5 x Retention - 10 x Retention])",
                                    "width": 152
                                },
                                {
                                    "key": "([Measures].[Actual Expense],[Measure Simulations].[Marketing Budget Simulation].[Marketing Budget Simulation].[10 x Retention])",
                                    "width": 133
                                },
                                {
                                    "key": "([Measures].[Actual Expense],[Measure Simulations].[Marketing Budget Simulation].[Marketing Budget Simulation].[5 x Retention])",
                                    "width": 142
                                },
                                {
                                    "key": "[Epoch].[Epoch].[Branch]",
                                    "width": 238
                                }
                            ],
                            "expansion": {
                                "automaticExpansion": true
                            }
                        }
                    }
                },
                "containerKey": "pivot-table"
            }
        },
        "height": 267
    }
```

**Comment by rgosse** [comment]
This was resolved with ActiveUI 4.3.9 (https://support.activeviam.com/jira/browse/UI-4286)
